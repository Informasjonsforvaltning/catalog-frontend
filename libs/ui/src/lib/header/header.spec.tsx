import React from 'react';
import { render } from '@testing-library/react';
import Header from '.';

describe('Header', () => {
  
  it('should render successfully', () => {
    const { baseElement } = render(<Header />);
    expect(baseElement).toBeTruthy();
    expect(baseElement).toMatchSnapshot();
  });
  
});
