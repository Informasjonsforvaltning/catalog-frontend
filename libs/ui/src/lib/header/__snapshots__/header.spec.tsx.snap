// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render successfully 1`] = `
<body>
  <div>
    <header
      class="sc-jSUZER eDxdhu"
    >
      <nav
        class="sc-gKPRtg gCazUl"
      >
        <a
          aria-label="Gå til hovedsiden"
        >
          <svgrurl
            class="sc-iBYQkv kerOCC"
          />
        </a>
        <ul
          class="sc-csuSiG hvkIuU"
        >
          <li
            class="sc-eDWCr lkyTcG"
          >
            <a
              href="https://fellesdatakatalog.digdir.no/guidance"
            >
              Registrere data
            </a>
          </li>
          <li
            class="sc-eDWCr lkyTcG"
          >
            <a
              href="https://admin.fellesdatakatalog.digdir.no"
            >
              Høste data
            </a>
          </li>
          <li
            class="sc-eDWCr lkyTcG"
          >
            <a
              href="https://datalandsbyen.norge.no"
            >
              Datalandsbyen
            </a>
          </li>
          <li
            class="sc-eDWCr lkyTcG"
          >
            <a
              href="https://data.norge.no"
            >
              Felles datakatalog
            </a>
          </li>
        </ul>
      </nav>
    </header>
  </div>
</body>
`;
